src/runtime/pay-flow.js

```diff
@@ -282,6 +287,11 @@ export class PayCompleteFlow {
       this.deps_
         .entitlementsManager()
         .pushNextEntitlements(response.entitlements.raw);
+      // Persist swgUserToken in local storage
+      if (response.swgUserToken) {
+        this.storage_ = new Storage(this.win_);
+        this.storage_.set(Constants.USER_TOKEN, response.swgUserToken, true);
```

Comment:
Up to you whether you want to do this now or later (when you add more sources for setting the user token, like from the entitlements response), but it might be good to have a shared helper file with functions that get and set the user token. As is, the logic for storing/retrieving the token from local storage using Constants.USER_TOKEN will be spread across different files; it would be nice if that logic was consolidated, even though it's relatively minimal.

Reply:
I can add that later. User token will be get in one place (call getEntitelments) and set in two places (SubscribeResponse and EntitlementResponse).